docs/source/clamav/clamav.rst

```diff
@@ -0,0 +1,287 @@
+
+Installation
+============
+
+ClamAV requires the NI Linux RT System Image version @tbd.X (build @tbd.0.0) or newer.
+
+Enabling/Disabling ClamAV Installation
+---------------------------------------
+
+ClamAV installation is controlled through the NILRT-SNAC configuration file. By default, ClamAV is
+**enabled** and will be automatically installed when you run ``nilrt-snac configure``.
+
+To disable ClamAV installation, edit ``/src/snac.conf`` and uncomment the disable line:
+
+.. code:: bash
+
+    # Uncomment to disable ClamAV installation
+    clamav = disabled      # ClamAV antivirus (verification only)
+
+To re-enable ClamAV (default behavior), comment out or remove the line:
+
+.. code:: bash
+
+    # clamav = disabled      # ClamAV antivirus (verification only)
+
+Automated Installation via NILRT-SNAC
+--------------------------------------
+
+The recommended method to install ClamAV on an NI Linux Real-Time system is through NILRT-SNAC,
+which handles the complete installation and configuration automatically:
+
+.. code:: bash
+    
+    nilrt-snac configure
+
+This automated installation process:
+
+- Installs ClamAV packages from pre-built IPK files
+- Configures virus database directories with proper permissions
+- Sets up logging with appropriate ownership and permissions
+- Restores DNS services if broken during installation
+- Creates the ``clamav-scan`` wrapper script with automatic swap file creation for scanning large files
+
+.. note::
+    The automated installation uses ``nilrt-snac configure`` which takes care of DNS service restoration,
+    and adds patches to create swap files for big file scanning.
+    This ensures ClamAV works reliably on resource-constrained embedded systems.
+
+Manual Installation (Alternative)
+----------------------------------
+
+If you prefer to install ClamAV manually, you can use opkg directly:
+
+.. code:: bash
+
+    opkg install clamav clamav-freshclam
+
+However, manual installation requires additional configuration steps that are automatically handled by
+NILRT-SNAC. The automated method is strongly recommended.
+
+Verifying Installation
+----------------------
+
+After installation, verify that ClamAV is properly installed and configured:
+
+**Quick Version Check:**
+
+.. code:: bash
+
+    clamscan --version
+
+You should see output showing the ClamAV version (1.4.3 or newer) and configuration paths.
+
+**Comprehensive Verification:**
+
+Use NILRT-SNAC's verify command to perform a complete system check:
+
+.. code:: bash
+
+    nilrt-snac verify
+
+This command checks:
+
+- ClamAV package installation status
+- Virus signature database integrity (checks for corrupted signatures)
+- Missing or incomplete signature files
+- Configuration file validity
+- Directory permissions and ownership
+- Service status and functionality
+
+@tbd : The verify command will report any issues found and provide guidance on how to resolve them.
```

Comment:
Let's remove this part from the main nilrt-docs. It's really just telling mainline NILRT customers - who probably aren't using SNAC - how to use SNAC.